frontend/src/pages/mentoringCreate/components/MentoringCreateForm/MentoringCreateForm.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from '@emotion/styled';
+
+import BaseInfo from '../BaseInfo/BaseInfo';
+
+function MentoringCreateForm() {
+  return (
+    <StyledContainer>
+      <BaseInfo></BaseInfo>
```

Comment:
p4: children으로 들어가는 값이 없다면 self-closing으로 바뀌어도 될 거 같아 😊
```suggestion
      <BaseInfo/>
```

Reply:
자동완성때문에 자꾸 컨벤션을 못 지키고 넘어가는 것 같아서 `self-closing 관련 lint 설정 추가`했어!

Reply:
[build: self-closing lint rule 추가](a2163f9b0ba66f977f26f7de64c5c797ce562323)
[feat: 셀프 클로징 태그로 변경](cbb4f41b68e860e7122394b28695d1a6d90b86d8)